dstep/driver/Application.d

```diff
@@ -37,20 +38,24 @@ class Application : DStack.Application
 
     private
     {
-        Language language;
+        Options defaultOptions;
```

Comment:
So I propose to rename it back to `blueprint`. It is hard to say how its end anyway, in the face of Dicebot's changes.